README.md

```diff
@@ -1,19 +1,27 @@
 # FFC Payment Batch Processor
 
-FFC service to process Siti Agri batch files and send individual transactions as messages.
+## Description
 
-## Prerequisites
+FFC microservice to process Siti Agri batch files and send individual transactions as messages.
```

Comment:
Maybe we should make clear that the messages are sent to an Azure Service Bus topic?

I also wonder if we should make reference to the fact we treat this service as part of an adapter for Siti Agri to help ensure that payments originating from there can be made